data/types/pumpstatus/pumpstatus.go

```diff
@@ -0,0 +1,103 @@
+package pumpstatus
+
+import (
+	"github.com/tidepool-org/platform/data"
+	"github.com/tidepool-org/platform/data/types"
+	"github.com/tidepool-org/platform/structure"
+)
+
+const (
+	Type = "pumpStatus"
+
+	MinPumpChargeRemaining = 0.0
+	MaxPumpChargeRemaining = 1.0
```

Comment:
Seems redundant (and also not used in the code)

Reply:
ok